### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by unidrelyn**
## Summary

Design is made to be responsive to sizes smaller than 1024px

## Screenshot 

image

#Trello card

https://trello.com/c/slVfYO9q/15-dise%C3%B1o-responsive

### CODE REVIEW

File: Maquetacion/scss/tech.scss [hunk 1]
```diff
@@ -12,35 +12,65 @@
...
+  }
+
+  .story {
+    @extend %text;
+    font-size: $text-medium;
+  }
+
+
+  @media (max-width: 1024px) {
+    grid-auto-columns: 1fr;
```
Comment (FrankIglesias) [inline Maquetacion/scss/tech.scss:54]: define the main class here you are applying these props

File: Maquetacion/scss/titles.scss [hunk 2]
```diff
@@ -13,4 +14,8 @@
     width: 70px;
     z-index: -10;
   }
+
+  @media (min-width: 1024px) {
```
Comment (FrankIglesias) [inline Maquetacion/scss/titles.scss]: same here

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: Maquetacion/scss/footer.scss [hunk 3]
```diff
@@ -25,3 +25,11 @@
     letter-spacing: 1px;
   }
 }
+
+@media (max-width: 1024px) {
+  .footer-bar {
```
Comment (FrankIglesias) [inline Maquetacion/scss/footer.scss]: .footer-bar .copy {

File: Maquetacion/scss/navbar.scss [hunk 4]
```diff
@@ -16,6 +16,15 @@
   }
 
   .menu {
+    display: none;
     font-size: $text-big;
   }
 }
+
+@media (min-width: 1024px) {
+  .nav-bar {
```
Comment (FrankIglesias) [inline Maquetacion/scss/navbar.scss]: same

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by jmtov — DISMISSED** [review]
Good job! 👍

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**